packages/edge/src/edge-headers.ts

```diff
@@ -68,6 +71,16 @@ function getHeader(request: Request, key: string): string | undefined {
   return request.headers.get(key) ?? undefined;
 }
 
+function getFlag(countryCode: string | undefined): string | undefined {
+  if (!countryCode) return undefined;
+  return String.fromCodePoint(
+    ...countryCode
+      .toUpperCase()
+      .split('')
+      .map(char => 127397 + char.charCodeAt(0))
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏name magic number

Can we pull `127397` into a named variable to make it clearer what it's used for? Is this the unicode offset before we get to the desired blocky letters?